src/actions/interfaces.ts

```diff
@@ -17,10 +18,11 @@ export enum ActionJsonKey {
 
 export type CommandActionParams = any [];
 export type ExecuteActionParams = any [];
-export type SnippetActionParams = any;
+export type SnippetActionParams = any | {data: any, service: IService} | {data: any};
```

Comment:
it can be service: any
or better to define something in here that will cover the service type